File: QUESTÃO 7.js
let userAge = Number(prompt('Enter your age to find out if you can ride a Hopi Hari roller coaster: '))
let userWeight = Number(prompt('Now enter your weight: '))

// I added the check to ensure that the user enters a number.
if (isNaN(userAge) || isNaN(userWeight)) {
    alert("Please enter valid numbers for age and weight.")
} else if (userAge >= 15 && userWeight < 120) {
    alert("Allowed!")
} else {
    alert("Prohibited!")
}

File: Python/QUESTÃO 7.py
userAge = int(input('Enter your age to find out if you can ride a Hopi Hari roller coaster: '))
userWeight = float(input('Now enter your weight: '))

if ((userAge >= 15) and (userWeight < 120)):
    print('Allowed!')
else:
    print('Prohibited!')